deployments/anvilprod/environment.py

```diff
@@ -991,6 +991,47 @@ def mkdict(previous_catalog: dict[str, str],
+    mksrc('bigquery', 'datarepo-ec751e53', 'ANVIL_ccdg_asc_ndd_daly_talkowski_brusco_asd_exome_20250514_ANV5_202505191354'), # noqa E501
+    mksrc('bigquery', 'datarepo-e880eb24', 'ANVIL_ccdg_asc_ndd_daly_talkowski_chung_asd_exome_20250514_ANV5_202505191404'), # noqa E501
+    mksrc('bigquery', 'datarepo-14218b2f', 'ANVIL_ccdg_asc_ndd_daly_talkowski_control_NIMH_asd_exome_20250514_ANV5_202505191412'), # noqa E501
+    mksrc('bigquery', 'datarepo-44dd3f7f', 'ANVIL_ccdg_asc_ndd_daly_talkowski_domenici_asd_exome_20250514_ANV5_202505191423'), # noqa E501
+    mksrc('bigquery', 'datarepo-b497e3bd', 'ANVIL_ccdg_asc_ndd_daly_talkowski_gargus_asd_exome_20250514_ANV5_202505191431'), # noqa E501
+    mksrc('bigquery', 'datarepo-23e0cf66', 'ANVIL_ccdg_asc_ndd_daly_talkowski_goethe_asd_exome_20250514_ANV5_202505191439'), # noqa E501
+    mksrc('bigquery', 'datarepo-dae963c2', 'ANVIL_ccdg_asc_ndd_daly_talkowski_gurrieri_asd_exome_20250514_ANV5_202505191448'), # noqa E501
+    mksrc('bigquery', 'datarepo-02a4023f', 'ANVIL_ccdg_asc_ndd_daly_talkowski_herman_asd_exome_20250514_ANV5_202505191456'), # noqa E501
+    mksrc('bigquery', 'datarepo-562d7351', 'ANVIL_ccdg_asc_ndd_daly_talkowski_hertz_picciotto_asd_exome_20250514_ANV5_202505191503'), # noqa E501
+    mksrc('bigquery', 'datarepo-90758277', 'ANVIL_ccdg_asc_ndd_daly_talkowski_hertz_picciotto_asd_wgs_20250514_ANV5_202505191512'), # noqa E501
+    mksrc('bigquery', 'datarepo-cd6d2aa6', 'ANVIL_ccdg_asc_ndd_daly_talkowski_hultman_asd_exome_20250514_ANV5_202505191519'), # noqa E501
+    mksrc('bigquery', 'datarepo-d21464d2', 'ANVIL_ccdg_asc_ndd_daly_talkowski_kolevzon_asd_exome_20250514_ANV5_202505191537'), # noqa E501
+    mksrc('bigquery', 'datarepo-186b5498', 'ANVIL_ccdg_asc_ndd_daly_talkowski_kolevzon_asd_wgs_20250514_ANV5_202505191546'), # noqa E501
+    mksrc('bigquery', 'datarepo-7fc43dc3', 'ANVIL_ccdg_asc_ndd_daly_talkowski_lattig_asd_exome_20250514_ANV5_202505191554'), # noqa E501
+    mksrc('bigquery', 'datarepo-e4ab9e05', 'ANVIL_ccdg_asc_ndd_daly_talkowski_mayo_asd_exome_20250515_ANV5_202505191602'), # noqa E501
+    mksrc('bigquery', 'datarepo-81d3c9c8', 'ANVIL_ccdg_asc_ndd_daly_talkowski_mcpartland_asd_exome_20250515_ANV5_202505191610'), # noqa E501
+    mksrc('bigquery', 'datarepo-0b020eb2', 'ANVIL_ccdg_asc_ndd_daly_talkowski_menashe_asd_exome_20250515_ANV5_202505191620'), # noqa E501
+    mksrc('bigquery', 'datarepo-f49b0d4c', 'ANVIL_ccdg_asc_ndd_daly_talkowski_minshew_asd_exome_20250515_ANV5_202505191632'), # noqa E501
+    mksrc('bigquery', 'datarepo-d0e9181a', 'ANVIL_ccdg_asc_ndd_daly_talkowski_parellada_asd_exome_20250515_ANV5_202505191640'), # noqa E501
+    mksrc('bigquery', 'datarepo-3965a084', 'ANVIL_ccdg_asc_ndd_daly_talkowski_passos_bueno_asd_exome_20250515_ANV5_202505191651'), # noqa E501
+    mksrc('bigquery', 'datarepo-c9d758bb', 'ANVIL_ccdg_asc_ndd_daly_talkowski_pericak_vance_asd_exome__20250515_ANV5_202505191658'), # noqa E501
```

Comment:
In a separate commit 

```suggestion
    bqsrc('c9d758bb', 'ccdg_asc_ndd_daly_talkowski_pericak_vance_asd_exome__20250515_ANV5_202505191658'),
```

and remove all the ugly noqa and formatter pragmas.